research-prototype/Programs/Source/ipae2e.mpc

```diff
@@ -18,7 +18,7 @@ if verbose:
     else:
         raise Exception(f"Unrecognized verbose value: {verbose}")
 
-if verbose and numrows_power > 5:
+if verbose and numrows_power > 9:
```

Comment:
2^9 is *a lot* of rows for verbose output. do we really need this?

Reply:
there is a bug in ipae2e.mpc when you go from 2^6 to 2^7 on `ipatestcase2.py`  I wasn't able to find it but was trying to print more to do so.